mmtbx/validation/clashscore.py

```diff
@@ -48,7 +48,7 @@ def as_table_row_phenix(self):
              abs(self.overlap) ]
 
   def __cmp__(self, other) : # sort in descending order
-    return cmp(self.overlap, other.overlap)
+    return libtbx.math_utils.cmp(self.overlap, other.overlap)
```

Comment:
Change the import to `from libtbx.math_utils import cmp` so nothing else needs to be changed.